assets/js/components/BatchEdit/PreviewItems.js

```diff
@@ -1,5 +1,68 @@
 import React from "react";
+import { Link } from "react-router-dom";
 
-export default function BatchEditPreviewItems() {
-  return <h2 className="title is-size-4">Preview of items go here</h2>;
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
+
+const inlineList = css`
```

Comment:
Is this code making the scroll bar purple?  If so, nice.  And if so can we make sure it works across all browsers relatively well?